src/styles/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  transition: transform 300ms;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__description {
+    width: 100%;
+    height: 36px;
+    margin: 40px 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.2);
+    .card__description {
+      color: #34568b;
+    }
+  }
+
+  &__product-code {
+    box-sizing: border-box;
+    width: 98px;
+    height: 14px;
+    margin: 0 0 16px;
+    color: $secondary-text-color;
+  }
+
+  &__grade {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
+    width: 100%;
+    margin: 0;
+  }
+
+  &__review {
+    background-position: 50%;
+    background-repeat: no-repeat;
+    color: $main-text-color;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    height: 18px;
+    width: 100%;
+    margin-top: 24px;
+    margin-bottom: 16px;
+
+    &-name {
+      text-align: center;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $secondary-text-color;
+      margin: 0;
+    }
+
+    &-cost {
+      text-align: center;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-text-color;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    border: none;
+    width: 100%;
+    height: 40px;
+    background-color: $blue-accent;
+    border-radius: 5px;
+    color: $white-accent;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    padding: 0;
+
+    &:hover {
+      background-color: $white-accent;
+      border: 1px solid $blue-accent;
+      color: $blue-accent;
```

Comment:
The hover effect for the button changes the background color and text color as per the task requirements. Ensure that these colors are consistent with your design specifications.